### PR TITLE
Add an installer script for enabling the simultor in a MATLAB online session

### PR DESCRIPTION
**Pull request by nunoguedelha**
Implements #21 .

### CODE REVIEW

**Comment by traversaro** [comment]
I guess that probably this script can be simplified once https://github.com/robotology/robotology-superbuild/issues/576 is solved, as in that case it would be sufficient to just `conda install` the simulator.

**Comment by nunoguedelha** [comment]
> I guess that probably this script can be simplified once [robotology/robotology-superbuild#576](https://github.com/robotology/robotology-superbuild/issues/576) is solved, as in that case it would be sufficient to just `conda install` the simulator.

Absolutelly. On the other hand, the current implementation can be the base of a template for installing any MATLAB code repo not yet added to the Conda packages, and having robotology dependencies.

**Comment by nunoguedelha** [comment]
Rebased.

File: README.md [hunk 1]
```diff
@@ -65,6 +65,32 @@ It is recommended to install these dependencies using the [robotology-superbuild
...
+  ```
+  system('curl -LO https://raw.githubusercontent.com/dic-iit/matlab-whole-body-simulator/featue/install-matlab-online/app/tools/matlabOnlineInstaller.m');
+  run matlabOnlineInstaller.
+  ```
+2. After step 1, you find the `matlab-whole-body-simulator` repository has been downloaded to `/MATLAB Drive/dev/matlab-whole-body-simulator`, further designated as <MATLAB_WB_SIM_SRC>, and its dependencies installed in `$HOME/miniforge3`. Select the robot model among the available ones in `<MATLAB_WB_SIM_SRC>/app/robots` and as described in the next section. For instance, selecting `iCubGenova04` would be done as follows:
+  ```
+  setenv('YARP_ROBOT_NAME','iCubGenova04');
+  ```
+3. Open the test model `<MATLAB_WB_SIM_SRC>/test_matlab_system_2020b.mdl`, and run it. After the compilation is complete, a figure will appear, displaying the robot performing the task driven by the controller used in the test model.
+
```
Comment (traversaro) [inline README.md:93]: The README contains already a lot of documentation that can confuse users that are just interested in using the library. Probably after https://github.com/robotology/robotology-superbuild/issues/576 we could move this "advanced" docs for people that want to modify matlab-whole-body-simulator in separate page, and just leave in the README the easy way of installing it?

**Review by traversaro — APPROVED** [review]
Minor comment just for the future.

File: README.md (hunk 1 above)
Comment (nunoguedelha) [reply README.md:93]: Yes absolutely, makes sense. In any case I would say that the steps 1-2-3 listed in the MATLAB Online section stay the same after https://github.com/robotology/robotology-superbuild/issues/576 is integrated. What changes is the content of the installer script. Then, the explanation on the bash limitations, along with the full sections "Installing from superbuild source" and "Installing from Robotology Conda channel" could be moved to the advanced docs as you mentioned.

Comment (nunoguedelha) [reply README.md:93]: Tracked this in https://github.com/dic-iit/matlab-whole-body-simulator/issues/34.

**Pull request merged**